solvebio/test/test_exports.py

```diff
@@ -20,6 +20,16 @@ def test_csv_exporter(self):
         self.assertTrue(path.isfile('/tmp/test.csv'))
         remove('/tmp/test.csv')
 
+    # FlatCSVExporter
+    def test_flat_csv_exporter(self):
+        dataset = Dataset.retrieve(self.TEST_DATASET_NAME)
+        query = dataset.query()[:10]
+
+        query.export('flat-csv', filename='/tmp/test_flat.csv')
+        self.assertTrue(path.isfile('/tmp/test_flat.csv'))
```

Comment:
I guess we might want to test that the flat export is flattening correctly by comparing against a correct file, maybe by MD5 or by content.